### PR TITLE
Make spatial features valid geojson

### PR DESCRIPTION
**Pull request by frankrowe**
## Status
**READY**

## Description
Should type of sc.spatial feature be `SPATIAL` or `FEATURE` (valid geojson)?

### CODE REVIEW

File: src/sc.spatialfeature.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ export function spatialFeature(storeId, obj) {
     properties: {},
     storeId: storeId,
     style: {},
-    type: 'SPATIAL'
+    type: 'FEATURE'
```
Comment (marcusthebrown) [inline src/sc.spatialfeature.js]: We probably want the type to be `Feature` (only the F is capitalized) which is what the spec defines as a valid value.

File: src/sc.spatialfeature.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ export function spatialFeature(storeId, obj) {
     properties: {},
     storeId: storeId,
```
Comment (marcusthebrown) [inline src/sc.spatialfeature.js]: Shouldn't all spatial features also have a `layerId` property?

Comment (frankrowe) [inline src/sc.spatialfeature.js]: should that be passed in the constructor along with storeId?

Comment (marcusthebrown) [inline src/sc.spatialfeature.js]: Yes sir!

**Comment by marcusthebrown** [comment]
👍

**Pull request merged**